src/containers/__snapshots__/Permission.spec.jsx.snap

```diff
@@ -41,7 +41,6 @@ exports[`Permission should display appName when query has been loaded 1`] = `
       class="MuiTypography-root MuiTypography-h2 MuiTypography-colorTextPrimary"
     >
       Permission: 
-      apps
```

Comment:
we lost one feature here 🤔

Reply:
Indeed, not sure what's going on I have to check

Reply:
It was a missing param in router mock @trollepierre

Reply:
fixed